OJP/OJP_Trips.xsd

```diff
@@ -604,24 +604,24 @@
 	</xs:complexType>
 	<xs:complexType name="PathGuidanceStructure">
 		<xs:annotation>
-			<xs:documentation> description of a piece of a TRIP. May include geographic information, turn instructions and accessibility information </xs:documentation>
+			<xs:documentation> description of a piece of a TRIP. May include geographic information, turn instructions and accessibility information.</xs:documentation>
```

Comment:
Leading space. "piece" of a trip? Maybe better "part"?

Reply:
changed